Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift

```diff
@@ -242,7 +248,7 @@ class EdgeHitProcessor: HitProcessing {
         }
 
         if !isValidPath(path) {
-            Log.error(label: self.SELF_TAG, "Dropping the overwrite path value: (\(String(describing: path))), since it contains invalid characters or is empty.")
+            Log.error(label: self.SELF_TAG, "Dropping the overwrite path value: (\(path ?? "")), since it contains invalid characters or is empty.")
```

Comment:
We might need to split isValidPath into two checks, we should only error out if there is an invalid path, but not for nil/empty as that is the default scenario.